tsconfig.cjs.json

```diff
@@ -2,6 +2,9 @@
     "extends": "./tsconfig.build.json",
     "compilerOptions": {
         "module": "CommonJS",
-        "target": "ES2015"
+        "target": "ES2020",
+        "resolveJsonModule": true,
+        "esModuleInterop": true,
+        "moduleResolution": "Node"
```

Comment:
other than module resolution and module, these should be inherited from the tsconfig.build.json config